TicketingSystem.Common/Model/Database/Payment.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TicketingSystem.Common.Model.Database
+{
+    public class Payment
+    {
+        public int PaymentId { get; set; }
+        public required DateTime PaymentTime { get; set; }
+
+        public Ticket Ticket { get; set; } = null!;
```

Comment:
Payment may contain more than one ticket

Reply:
Payments are now linked to Carts